### ISSUE TITLE
Improve test for sheafification 

### ISSUE THREAD

**Issue by mohamed-barakat** [homalg-project/ToricVarieties_project#75]
There are other simpler tests for "does sheafify to zero" which can be added before going for the final test [here](https://github.com/homalg-project/ToricVarieties_project/blob/7dc7c5af027e4d9e52b3032ac7d77cb97a7e8a2a/CoherentSheavesOnToricVarieties/gap/CoherentSheaves.gi#L60). One of them is to test the dimension of the module using the command `AffineDimension`.

**Comment by HereAround** [comment]
Thank you @mohamed-barakat. Is `AffineDimension` already supported for the FPGradedModules via `FreydCategories`?

If not, I support a conversion amog the f.p. graded module implementation in `GradedModules` and those in `FreydCategories` in https://github.com/homalg-project/ToricVarieties_project/blob/master/ToolsForFPGradedModules/gap/Conversion.gd. One way to implement your suggestion is thus to overload this (and if needed more) functionality.

**Comment by mohamed-barakat** [comment]
No, in the case of left modules you have to extract the matrix of relations and apply `AffineDimension` to it. In the case of right modules, you have to transpose it first before applying `AffineDimension`.

I am now working on a package called `IntrinsicGradedModules` which should at some point replace `GradedModules` but is now based on `FPGradedModules` and in particular CAP-based. This package will include some enhancements for `FPGradedModules`. Once the package is stable I will make PRs to `FreydCategoriesForCAP` including these enhancements.

**Comment by HereAround** [comment]
Thank you for elaborating @mohamed-barakat. The structure makes sense to me. However, this will take time. Furthermore, currently the sheafification is merely a "container" for `FPGradedModules`. None of the implemented algorithms benefits from it. I hope this changes at some point in the future...

Consequently, to me this seems not a high priority at this point. Please keep me posted. Once you have the relevant structures/packages ready, I am happy to address this.
